src/pages/gateway/create-mesh.md

```diff
@@ -189,6 +189,36 @@ aio api-mesh:update update-mesh.json
     }
 ```
 
+## Modify projects and workspaces
+
+<InlineAlert variant="info" slots="text"/>
+
+When creating a mesh for the first time, you must select the project and workspace that you want to create the mesh in. Alternatively, you can use the following [aio commands](https://github.com/adobe/aio-cli#commands) to select or deselect a project or workspace without interacting with your mesh.
+
+[aio commands](https://github.com/adobe/aio-cli#commands) allow you to list the current selections, manually select a project or workspace, or remove the currently selected project or workspace from the cache.
+
+### View cached project and workspace
+
+To see your current cache configuration, use the [`aio config:get:console`](https://github.com/adobe/aio-cli#aio-configget-key) command, which includes the currently selected organization, project, and workspace.
+
+### Select a project or workspace
+
+If you want to change the selected project, use the [`aio console:project:sel`](https://github.com/adobe/aio-cli#aio-consoleprojectsel-projectidorname) or [`aio console:project:select`](https://github.com/adobe/aio-cli#aio-consoleprojectselect-projectidorname) commands followed by the name or id of the project. You will be prompted to select a project.
```

Comment:
Just mention one form of these commands, preferably the one with the final verb spelled out (select or delete). 

This section and the next need some restructuring. 

If you run `aio console:project:select` you have to use the arrow keys and Return to select a project. Ditto for the `workspace:select` command.

If you run either of these commands and specify an ID or name, then there are no prompts.

Rework the next paragraph  and include some examples, like in the Delete section below.

Reply:
[This page](https://developer.adobe.com/app-builder/docs/getting_started/first_app/#2-creating-a-new-project-on-developer-console) says:

`Each App Builder project has two default workspaces: Production and Stage. You can add more workspaces as needed. The Production workspace is special, as it is used for the submission and distribution flow. When you’re ready to deploy your app, you will submit it for approval from the Production workspace.`

So maybe I should tell them to use `Stage` when in doubt.

Reply:
Yeah, App Builder is the distinction. I was thinking I created a project early on that had a different number of workspaces (maybe 0). Definitely steer them to Stage. Maybe we both need to describe setting up a Develop workspace.

Reply:
After testing this out on other `projects` and talking with @revanth0212, it makes more sense to only mention the selection method, because the name/id is not human-readable and is assigned to them.